bokeh/sphinxext/_templates/gallery_detail.rst

```diff
@@ -3,6 +3,8 @@
 .. index::
    single: examples; {{ filename }}
 
+{{ ref }}
+
```

Comment:
I added a reference to all the examples  in the syntax `_example_{name}_{model/plotting}`. Not sure if this is helpful. I wanted to use it for the references of the gallery images, but it was not working. If you think this is to much, this can be removed again.